README.md

```diff
@@ -236,7 +236,6 @@ Hurray! You successfully made a contribution!
   - [Evandro Rodrigues](https://github.com/evnrodr)
 
 - ### **F**
-  
```

Comment:
I am so sorry can this line be added on I was confused how you removed a line earlier and then added a new line. Thank you

Reply:
now is it okay or need change?? @CodingSpecies

Reply:
That line needs adding on again and that is it after that.

Reply:
I don't understand what are you saying . I am closing this PR and creating new so please review it.